include/wx/control.h

```diff
@@ -46,7 +46,8 @@ class WXDLLIMPEXP_CORE wxControlBase : public wxWindow
 
     // get the control alignment (left/right/centre, top/bottom/centre)
     int GetAlignment() const { return m_windowStyle & wxALIGN_MASK; }
-
+    void MarkAvailable(bool available) { m_available = available; }
+    bool IsAvailable() const { return m_available; }
```

Comment:
We shouldn't need to add functions that only make sense for toolbar tools to base `wxControl` class. You will have to change the code to use them in `wxToolBatTool` only.